src/applications/personalization/appointments/actions/index.js

```diff
@@ -164,7 +164,7 @@ export function fetchConfirmedFutureAppointments() {
             ],
           });
           recordEvent({
```

Comment:
I noticed the usage of `isClientError(error.status)` and 'isServerError` in some of the other handling. If that something that also needs to be checked here, or would this always result in a server error?